src/index.js

```diff
@@ -57,14 +57,30 @@ export default class ScrollBehavior {
     });
   }
 
+  getSavedScrollTarget(key) {
```

Comment:
Can you make this a separate PR?

Reply:
I need it in the same PR to be able to test the new methods.